include/header.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+function echoHeader($pageTitle){
+    echo"
+   </head>
+   <title>".$pageTitle."</title>
+    </head> 
+    <link rel='stylesheet' href=<link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.7.0/css/all.css' integrity='sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ' crossorigin='anonymous'>
+
+    <link rel='stylesheet' type='text/css' href='style.css'/>
+    <body>
+    <div class='parent'>    
+        <h1 class='child'>COURTYARD COLLECTIVE</h1>
+        <a href='signup.php'><div class='child p1 rightborder '> Sign up</div></a>
+        <a href='login.php'><div class='child p1 '> Login</div></a>
+    </div>
+        <div class='dropdown'>
+        <button class='dropbtn'><i class='fas fa-bars'></i></button>
+        <div class='dropdown-content'>
+        </div>
+    </div>
+    
+        <div class='topnav;topnav a; topnav a:hover;'>
```

Comment:
Normally you separate class names with spaces, no semicolons. And `topnav a` implies you're trying to style a link, but this isn't a link. I've never seen this syntax before. Is it working the way you want? Maybe this is some crazy new html/css I'm not familiar with. If that's the case, would you mind sending me a link to wherever you learned it (I'd like to understand what's going on here)